index.html

```diff
@@ -224,27 +230,46 @@ <h4 id="about-intro">About me</h4>
           </div>
         </div>
       </section>
-      <!-- contact setion -->
-      <section id="contact-sections">
+      <!-- contact section -->
+      <footer id="contact-sections">
         <div id="contact-intro">
           <h4>
             I'm always interested in hearing about new projects, so if you'd
             like to chat please get in touch.
           </h4>
         </div>
-        <form action="action" id="personal-info">
-          <input type="text" placeholder="Full name" id="fname" name="fname" />
+        <form
+          action="https://formspree.io/f/mbjwwwvr"
+          method="post"
+          id="personal-info"
+        >
+          <input
+            type="text"
+            placeholder="Full name"
+            id="fname"
+            name="fname"
+            maxlength="30"
+            required
+          />
           <input
             type="email"
             id="email"
             placeholder="Email address"
             name="email"
+            required
           />
-          <textarea name="message" id="message" cols="30" rows="10">
+          <textarea
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          >
           Enter text here</textarea
           >
```

Comment:
**\[OPTIONAL\]**
- [ ] I suggest you make the "Enter text here" a placeholder instead